src/openrct2/audio/Audio.cpp

```diff
@@ -71,27 +71,27 @@ static int32_t SoundVolumeAdjust[SOUND_MAXID] =
     0,      // SOUND_SCREAM_4
     0,      // SOUND_SCREAM_5
     0,      // SOUND_SCREAM_6
-    0,      // SOUND_LIFT_3
+    0,      // SOUND_LIFT_VEKOMA
```

Comment:
Hm, why Vekoma? Isn't this sound also used for the Junior RC and dinghy slide?

Reply:
Yes it is, though the sound is used in odd places anyway. Per discussion, I'll rename it FRICTION_WHEEL for now.

Reply:
`FRICTION_WHEELS`, they always come in pairs ;)